<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>niftk::VLVideoOverlayWidget</class>
 <widget class="QWidget" name="niftk::VLVideoOverlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="m_LeftImageCheckBox">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_LeftImageCombo"/>
       </item>
       <item>
        <widget class="QCheckBox" name="m_RightImageCheckBox">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>right</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_RightImageCombo"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="m_TrackingLabel">
         <property name="text">
          <string>tracking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QmitkDataStorageComboBoxWithSelectNone" name="m_TrackingCombo"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="m_OpacityLabel">
         <property name="text">
          <string>opacity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="m_OpacitySlider">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_TrackedViewCheckBox">
         <property name="text">
          <string>tracked</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_3DViewCheckBox">
         <property name="text">
          <string>3D</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="m_Splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBoxWithSelectNone</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBoxWithSelectNone.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
